tools/torch_quant/tests/test_quantizer.py

```diff
@@ -21,7 +21,7 @@
 class QuantizerTest(unittest.TestCase):
```

Comment:
Maybe the `tools` dir is unnecessary, how about move `torch_quant` to the top-level dir?

Reply:
There will be other tools, like sparsification,  tf quant and so on. It could be kind of messy to put all these under root?

Reply:
how many tools are on going? Do we want to package these tools into one wheel package or some individual package?

Reply:
torch-comp ? tf-comp? :P

Reply:
I would prefer separated pkgs for different tools, so we can handle deps and release separately. At least, torch and tf tools should be seperated.